terminology-guides/spanish-terminology.md

```diff
@@ -4,57 +4,62 @@
+| Sweep Unmixable | Barrido no se puede mezclar or Limpiar lo inmezcable | A contributor suggested this translation. Open for discussion. |
+| Tail Emission | Emisión Final / Definitiva / Continua | Open to discussion, "emisión de cola" or "emisión trasera" could be misunderstood. |
 | Testnet | Red de Pruebas | |
-| Thread | Hilo, subproceso, núcleo | "Hilo" is not a common word in several geopgrphical zones but found this inside [Intel](https://ark.intel.com/es/products/72164/Intel-Core-i5-3230M-Processor-3M-Cache-up-to-3_20-GHz-rPGA) software. Cantidad de subproceso = Un hilo, o hilo de ejecución, es un término de software para la secuencia de instrucciones de orden básico que puede pasar por o procesarse en un núcleo de CPU individual. Cantidad de núcleos = Núcleos es un término de hardware que describe el número de unidades de procesamiento independientes en un componente computacional individual (matriz o chip). |
-| Tuning | Configurar/Ajuste/Optimización | Technical terms for software language, any of these could be used, depends on the linguistic structure. |
-| Untraceable | No rastreable | It is not recommended to use "imposible de rastrear", nothing is "imposible" to trace, technology has flaws or could be broken. |
-| View Key | Clave de visualización | |
-| Wallet | Monedero | We used "monedero" because it is already being used inside the Bitcoin community |
-| Wallet creation height | Altura de creación de cartera | |
-| White paper | Libro blanco | Wikipedia defines the term as "Libro blanco" |
+| Thread | Hilo / Subproceso / Núcleo | The word "hilo", is not a common word in every Spanish native region but found this inside an [Intel](https://ark.intel.com/es/products/72164/Intel-Core-i5-3230M-Processor-3M-Cache-up-to-3_20-GHz-rPGA) article. Cantidad de subproceso = Un hilo, o hilo de ejecución, es un término de software para la secuencia de instrucciones de orden básico que puede pasar por o procesarse en un núcleo de CPU individual. Cantidad de núcleos = Núcleos es un término de hardware que describe el número de unidades de procesamiento independientes en un componente computacional individual (matriz o chip). |
+| Tuning | Configurar / Ajuste / Optimización | Technical terms for software language, any of these could be used, depends on the linguistic structure. |
+| Untraceable | No Rastreable | It is not recommended to use "imposible de rastrear", nothing is "imposible" to trace, technology has flaws or could be broken. |
+| View Key | Clave de Visualización | |
+| Wallet | Monedero | We used "monedero" because it is already being used inside the Bitcoin community. |
+| Wallet Creation Height | Altura de creación de cartera | |
+| White Paper | Libro Blanco | Wikipedia defines the term as "Libro blanco". |
 
 
 ## Not translated
 
 | **Inglés** | **Español** | **Notes** |
 | --- | --- | --- |
-| blob | --- | --- |
-| nonce | --- | --- |
+| Blob | --- | No translation |
```

Comment:
This word should be keep this way as someone suggested, not remember who or where, but since there isn't a direct translation we could use. The same is for _Nonce_.

Reply:
So should I add "No translation" in the "notes"?